### PR TITLE
Add email property to the user

### LINKED ISSUE #1: Q-1 - Feature: Add email to the user entity

### PR DESCRIPTION
**Pull request by popsorin**
Closes #1

### CODE REVIEW

File: src/Entity/User.php [hunk 1]
```diff
@@ -63,4 +76,20 @@ public function getName(): ?string
...
+     */
+    public function getEmail(): string
+    {
+        return $this->email;
+    }
+
+    /**
+     * @param string $email
+     */
+    public function setEmail(string $email): void
```
Comment (calinpristavu) [inline src/Entity/User.php]: usually setters should be fluid (aka `return $this;`):

File: public/views/assets/admin-user-details.phtml [hunk 2]
```diff
@@ -56,10 +56,10 @@
                 <label for="userName">Name</label>
                 <input type="text" name="name" value="<?php echo $user->getName() ?>" class="form-control" id="userName" placeholder="<?php echo $user->getName() ?>">
             </div>
-<!--            <div class="form-group">-->
-<!--                <label for="userEmail">Email</label>-->
-<!--                <input type="email" name="email" class="form-control" id="userEmail" placeholder="User email">-->
-<!--            </div>-->
+            <div class="form-group">
+                <label for="userEmail">Email</label>
+                <input type="text" name="email" value="<?php echo $user->getEmail() ?>" class="form-control" id="userEmail" placeholder="<?php echo $user->getEmail() ?>">
```
Comment (noonlit) [inline public/views/assets/admin-user-details.phtml]:
```suggestion
                <input type="email" name="email" value="<?php echo $user->getEmail() ?>" class="form-control" id="userEmail" placeholder="<?php echo $user->getEmail() ?>">
```

Comment (noonlit) [inline public/views/assets/admin-user-details.phtml]: Does this input need a placeholder?

File: public/views/assets/admin-user-details.phtml [hunk 3]
```diff
@@ -82,10 +82,10 @@
                     <label for="userName">Name</label>
                     <input type="text" name="name" class="form-control" id="userName" placeholder="name">
                 </div>
-<!--                <div class="form-group">-->
-<!--                    <label for="userEmail">Email</label>-->
-<!--                    <input type="email" name="email" class="form-control" id="userEmail" placeholder="User email">-->
-<!--                </div>-->
+                <div class="form-group">
```
Comment (noonlit) [inline public/views/assets/admin-user-details.phtml]: Is there a way to avoid all this duplicate code in the template?

**Pull request merged**